src/utils/fhir.js

```diff
@@ -186,7 +197,29 @@ function subscriptionsFromBundle(specBundle, url, token) {
     throw new Error('Specification Bundle does not contain a PlanDefinition');
   }
 
-  return planDefs.map(planDef => subscriptionsFromPlanDef(planDef.resource, url, token)).flat();
+  const subscriptions = planDefs
+    .map(planDef => subscriptionsFromPlanDef(planDef.resource, url, token))
+    .flat();
+
+  subscriptions.forEach(async subscription => {
+    const subscriptionId = subscription.id;
+
+    // Store subscriptions in database
+    debug(`  ...Subscription/${subscriptionId}`);
```

Comment:
One more thing, can this be restructured. It makes it look like this Subscription was fetched from the server but it wasn't

```
  medmorph-backend:server Fetched http://localhost:8090/fhir/Bundle/7f06514d-7d21-4635-b488-82fc2aa7af76 +184ms
  medmorph-backend:server   ...PlanDefinition/PD1 +0ms
  medmorph-backend:server   ...PlanDefinition/plandefinition-cancer-example +0ms
  medmorph-backend:server   ...Subscription/subPD1encounter-close +1ms
  medmorph-backend:server   ...Subscription/subplandefinition-cancer-exampleencounter-close
```

Reply:
I can change it to something like `Saved Subscription with id ${subscriptionId} to db.` or should I just remove the debug statement since there's one right below for posting to ehr server?

Reply:
I would keep the debug because its nice to have the log. Your suggestion sounds good, or even just replace `  ...` with `Saved `